src/reflection.js

```diff
@@ -1,7 +1,22 @@
 import { filter } from 'funcadelic';
 
+const { getPrototypeOf, getOwnPropertyDescriptors, assign } = Object;
+
 export function methodsOf(Type) {
   return filter(({ key: name, value: desc }) => {
     return name !== 'constructor' && typeof name === 'string' && typeof desc.value === 'function';
-  }, Object.getOwnPropertyDescriptors(Type.prototype));
+  }, getPrototypeDescriptors(Type));
 }
+
+export function getPrototypeDescriptors(Type) {
+  let prototype = getPrototypeOf(Type);
+  // do not include Object.prototype descriptors
+  if (prototype && prototype !== getPrototypeOf(Object)) {
```

Comment:
Right, but:
```
> Object.getPrototypeOf(Object) === Object.prototype
false

> Object.getPrototypeOf(Object) === Function.prototype
true

```

In other words, are you sure `getPrototypeOf(Object)` and `getPrototypeOf(Type)` are doing what you think they're doing?